src/cert.rs

```diff
@@ -173,6 +173,22 @@ impl<'a> Cert<'a> {
         self.subject.as_slice_less_safe()
     }
 
+    /// Get the RFC 5280-compliant [`SubjectPublicKeyInfo`] (SPKI) of this [`Cert`].
+    #[cfg(feature = "alloc")]
+    pub fn subject_public_key_info(&self) -> SubjectPublicKeyInfo {
+        // Wrap the SPKI contents back into a proper ASN.1 SEQUENCE
+        SubjectPublicKeyInfo::from(der::asn1_wrap(
+            Tag::Sequence,
+            self.subject_public_key_info_contents(),
+        ))
+    }
+
+    /// Raw DER encoded SubjectPublicKeyInfo (SPKI).
+    /// Contains only the content of the RFC 5280 SEQUENCE, not the full encoding.
+    pub(crate) fn subject_public_key_info_contents(&self) -> &[u8] {
+        self.spki.as_slice_less_safe()
+    }
```

Comment:
I'm not sure there's a lot of value in having this fn instead of having the `pub fn subject_public_key_info()` use `self.spki.as_slice_less_safe()` directly in the `der::asn1_wrap` call.

Reply:
Agreed—removed this in [ef13aaa](https://github.com/rustls/webpki/pull/253/commits/ef13aaa72d8377055b4b56406ab63ac774e2b358).

Reply:
Great thanks! I think the PR description will also need an update to reflect this change.